examples/apigw.yml

```diff
@@ -11,7 +11,7 @@ discovery:
           statistics: [Average, Maximum, p95, p99]
         - name: Count
           statistics: [SampleCount, Sum]
-        - name: 4xx
+        - name: 4XXError
           statistics: [Sum]
-        - name: 5xx
+        - name: 5XXError
```

Comment:
@vasyharan I think `4xx` and `5xx` are valid metrics for APIGateway v1 (http), while `4XXError` and `5XXError` are for APIGateway v2 (rest). Maybe we should have two example files?

Reply:
I didn't realize that these were different versions! Updated to add a new files for v2.